.tests/integration/config/config.yaml

```
@@ -214,6 +214,12 @@ manta_config_n:
 manta_run_workflow_n:
   container: "docker://hydragenetics/manta:1.6.0"
 
+melt:
+  container: "reference/MELT_v2.2.2.sif"
+  bed: "reference/Hg38.genes.bed"
+  extra: ""
```

```suggestion
  extra: "-r 150"
```
Approximate read length of the supplied bam files.